orangecontrib/text/datasets/url.tab

```diff
@@ -0,0 +1,19 @@
+topic	url_case_study	url

```

Comment:
can you suggest changes [here](https://docs.google.com/spreadsheets/d/1yzcXo2m-aTgkWzUigeti2zBHTQp8io9LdkY3DCrPGs8/edit?usp=sharing) ?
That would make the work easier